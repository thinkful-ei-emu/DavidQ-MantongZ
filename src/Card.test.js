import React from 'react';
import renderer from 'react-test-renderer';

import ReactDOM from 'react-dom';
import Card from './Card';

describe('<Card />', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<Card />, div);

        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders UI as Expected',()=>{
      const tree = renderer.create(<Card />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
