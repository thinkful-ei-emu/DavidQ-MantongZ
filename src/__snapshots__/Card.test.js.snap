// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders UI as Expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
